tests/openwisp2/settings.py

```diff
@@ -148,7 +148,7 @@
         },
         'py.warnings': {'handlers': ['console'], 'propagate': False},
         'celery': {'handlers': ['console'], 'level': 'DEBUG'},
-        'celery.task': {'handlers': ['console'], 'level': 'DEBUG'},
+        #'celery.task': {'handlers': ['console'], 'level': 'DEBUG'},
```

Comment:
Since we are using git, you can just remove this line (instead of commenting out). It's possible to revert if required.